certified-connectors/tyntec-WA/apiDefinition.swagger.json

```diff
@@ -504,16 +508,2532 @@
 			  "description": "default",
 			  "schema": {
 				"type": "array",
-				"items": {}
+				"items": {
+				  "type": "object",
+				  "properties": {
+					"channel": {
+					  "type": "string",
+					  "description": "channel"
+					},
+					"scopes": {
+					  "type": "array",
+					  "items": {
+						"type": "string"
+					  },
+					  "description": "scopes"
+					},
+					"id": {
+					  "type": "object",
+					  "properties": {
+						"phoneNumber": {
+						  "type": "string",
+						  "description": "phoneNumber"
+						}
+					  },
+					  "description": "id"
+					},
+					"callback": {
+					  "type": "object",
+					  "properties": {
+						"inboundMessageUrl": {
+						  "type": "string",
+						  "description": "inboundMessageUrl"
+						}
+					  },
+					  "description": "callback"
+					}
+				  }
+				}
 			  }
 			}
 		  },
 		  "x-ms-visibility": "internal",
 		  "operationId": "TestMyAPIConnection",
-		  "description": "TestConnection",
+		  "description": "Tests your API connection",
 		  "summary": "testConnection",
 		  "parameters": []
 		}
+	  },
+	  "/conversations/v3/messages/{messageId}/status": {
+		"get": {
+		  "responses": {
+			"200": {
+			  "description": "default",
+			  "schema": {
+				"type": "object",
+				"properties": {
+				  "messageId": {
+					"type": "string",
+					"description": "messageId"
+				  },
+				  "status": {
+					"type": "string",
+					"description": "status"
+				  },
+				  "timestamp": {
+					"type": "string",
+					"description": "timestamp"
+				  }
+				}
+			  }
+			}
+		  },
+		  "summary": "Status Check",
+		  "description": "Check for status of your message",
+		  "operationId": "StatusCheck_v3",
```

Comment:
@ridlees all operationIds should be in Camel case please change to "StatusCheckV3". please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Please fix all operationIds in your swagger file.

Reply:
Fixed